docker-compose.yml

```
@@ -61,4 +80,5 @@
     command: celery -A config.celery_app beat -l info
 
 volumes:
-  postgres_data:
+  postgres_data:
+  minio_data:
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)